backend/clickhouse/trace_row.go

```diff
@@ -7,6 +7,15 @@ import (
 	"github.com/google/uuid"
 )
 
+const TraceAttributeValueLengthLimit = 2 << 15
+
+func truncateValue(input string) string {
+	if len(input) > TraceAttributeValueLengthLimit {
+		return input[:TraceAttributeValueLengthLimit] + "..."
```

Comment:
should we log the project id here? i guess it will be obvious which trace attributes are truncated because of the `...` so perhaps it's not worth adding the large log volume